src/makeCalculator.js

```diff
@@ -41,27 +41,27 @@ function makeCalculator() {
     result: 0,
 
     reset() {
-      this.result = 0;
+      calculator.result = 0;
 
-      return this;
+      return calculator;
     },
     add(number) {
-      this.result += number;
+      calculator.result += number;
     },
     subtract(number) {
-      this.result -= number;
+      calculator.result -= number;
     },
     divide(number) {
-      this.result /= number;
+      calculator.result /= number;
     },
     multiply(number) {
-      this.result *= number;
+      calculator.result *= number;
     },
 
     operate(operation, number) {
-      operation.bind(this, number)();
+      operation(number);
 
-      return this;
+      return calculator;
```

Comment:
you can still return this here, that was fine

Reply:
yes I know, but I used ctrl + d to target all "this" occurances and that is why it got changed as well. Is it neccessary to switch it back to this? Isn't it basically the same?

Reply:
It's true, returning either `calculator` or `this` from the makeCalculator function will both return the same object. One small thing is that if you change the name in the future you won't have to refactor it in all places, `this` will always refer to the same.

There is also a thing thqt returning `calculator` and `this` is different in the way the object is accessed after the makeCalculator function is called. If `this` is returned, the object will be accessed through the variable or context in which the function was called

It's a subtle difference but it's better to keep it that way.

Reply:
okay :D